scripts/loadOrgData.js

```diff
@@ -49,3 +49,16 @@ if (fs.existsSync(path.join(dataRepoPath, "config/assets"))) {
     { recursive: true },
   );
 }
+const additionalFileSrc = path.join(dataRepoPath, "config/opengraph-image.tsx");
+const additionalFileDest = path.join(
+  cwd,
+  "app/contributors/[slug]",
+  "opengraph-image.tsx",
+);
+
+if (fs.existsSync(additionalFileSrc)) {
+  console.log("Copying additional file opengraph-image.tsx...");
+  fs.copyFileSync(additionalFileSrc, additionalFileDest);
+} else {
+  console.error("File opengraph-image.tsx not found in the config folder");
+}
```

Comment:
Why copy the typescript file through load org data? It's not part of the data-repo right?